int32 grasp
float32 rb_x
float32 rb_y
float32 rb_theta
float32 obj_x
float32 obj_y
float32 obj_theta

